data/i18n/pt-br/pt-br.toml

```diff
@@ -194,38 +334,20 @@ other = "Objetivos"
 [options_heading]
 other = "Opções"
 
+[outdated_blog__message]
```

Comment:
There are two `_` is that ok?

Reply:
Hi there, @MrErlison! 👋 

Yes, the English key also looks like that: https://github.com/kubernetes/website/blob/13a9d24f778372fece6b686300c6bbcbcdd8a6ed/data/i18n/en/en.toml#L337-L338

I can also see that it renders correctly in Portuguese in my preview
(https://deploy-preview-41546--kubernetes-io-main-staging.netlify.app/pt-br/blog/2020/09/02/scaling-kubernetes-networking-with-endpointslices/):
![image](https://github.com/kubernetes/website/assets/698465/c948b436-cfc1-4099-af8d-bbcf069d549c)